carbonserver/carbonserver/api/infra/database/sql_models.py

```diff
@@ -41,6 +41,7 @@ class Run(Base):
     experiment_id = Column(UUID(as_uuid=True), ForeignKey("experiments.id"))
     os = Column(String, nullable=True)
     python_version = Column(String, nullable=True)
+    codecarbon_version = Column(String, nullable=True)
```

Comment:
I'm wondering if we need a more generic metadata, like:
- `client_type` to store `codecarbon package`
- `client_version` to store `2.1.3`
But maybe nobody else will use CodeCarbon API to store data.
@SaboniAmine Any idea ?

Reply:
As we only have data retrieved from the package client at the moment, I guess a single column is enough.
If needed we could change this in the future.

Reply:
Or we could have a column `metadata` and store the metadata as a dictionary?

Reply:
It will be difficult to query it, no ? I prefer to keep specific field, but it's a good idea if we want to be open to custom field.